test/extended/testdata/build-extended/bc-scripts-in-the-image.json

```diff
@@ -0,0 +1,34 @@
+{
+  "kind": "BuildConfig",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "java-extended-build-from-image"
+  },
+  "spec": {
+    "source": {
+      "git": {
+        "uri": "https://github.com/php-coder/java-maven-hello-world.git",
+        "ref": "s2-assemble-only"
+      }
+    },
+    "strategy": {
+      "sourceStrategy": {
+        "from": {
+          "kind": "ImageStreamTag",
+          "name": "wildfly:10.0",
+          "namespace": "openshift"
+        },
+        "runtimeImage": {
+          "kind": "DockerImage",
+          "name": "phpcoder/hello-world-runtime:0.2"
```

Comment:
>Ok so you need two images, one that has the mapping annotation and one that doesn't.. i'd still expect them to be otherwise identical (one can even extend the other). using two totally separate images (one php, one jdk) makes it confusing and harder to maintain the tests.

I agree with you. That's why `phpcoder/hello-world-runtime` image extends `phpcoder/base-jdk`.